text/0071-trailing-commas.md

```diff
@@ -0,0 +1,170 @@
+# Trailing Commas
+
+## Related issues and PRs
+
+- Reference Issues: https://github.com/cedar-policy/cedar/issues/1024
+- Implementation PR(s):
+
+## Timeline
+
+- Started: 2024-06-26 
+
+## Summary
+
+The Cedar grammar (both policies and schemas) should accept trailing
```

Comment:
Human-readable schemas do allow trailing commas.

Reply:
They do in some places, not in others. This proposes to make it so it's allowed in every possible spot.